include/wx/gtk/mimetype.h

```diff
@@ -12,18 +12,20 @@
 
 #include "wx/defs.h"
 
-#if defined(__UNIX__)
-#include "wx/unix/mimetype.h"
-#elif defined(__WINDOWS__)
-#include "wx/msw/mimetype.h"
+#if defined(__WINDOWS__)
```

Comment:
This might affect Cygwin, where I think both `__WINDOWS__` and `__UNIX__` are defined. I'm not really sure if anybody still uses it, but it looks like we could just prepend `__DARWIN__` test to the existing code and be sure not to break anything else.

Reply:
I put it this way because it imitates the order in `mimecmn.cpp` line 44ff and I think it would be good to have these two in synch

Reply:
Sorry, I'm not really sure, I just thought it would be best to minimize changes to this (very) rarely used build configuration to avoid breaking it. Maybe we should actually just declare Cygwin unsupported, I don't know how much sense does it make to still use it now, with MSYS2 and WSL being also available.